optuna/study.py

```diff
@@ -1104,11 +1105,14 @@ def objective(trial):
 
     if len(directions) < 1:
         raise ValueError("The number of objectives must be greater than 0.")
-    elif any(d != "minimize" and d != "maximize" for d in directions):
+    elif any(
+        d not in ["minimize", "maximize", StudyDirection.MINIMIZE, StudyDirection.MAXIMIZE]
+        for d in directions
+    ):
         raise ValueError("Please set either 'minimize' or 'maximize' to direction.")
```

Comment:
A nit, but how about adding `StudyDirection` to the error message?
```suggestion
    ):
        raise ValueError(
            "Please set one of 'minimize', 'maximize' or a StudyDirection member to direction."
        )
```

Reply:
Thanks for your suggestion! I updated the error message.

Reply:
Thank you for your update. It's better than my proposal 👍